frontend/website/docs/getting-started.md

```diff
@@ -0,0 +1,80 @@
+# Getting Started
+
+This section will walk you through the requirements needed to run a Coda protocol node on your local machine and connect to the network.
+
+!!! note
+    Last updated for release v0.0.1
+
+## Requirements
+
+**Software**: macOS (10.x.x and above) or Linux (currently supports Debian 9 and Ubuntu)
+
+**Hardware**: Sending and receiving coda does not require any special hardware, but participating as a node operator currently requires:
+- at least a 4-core processor
+- at least 8 GB of RAM
+
+GPU's aren't currently necessary, but may be required for node operators when the protoctol is upgraded.
+
+**Network**: At least 1 Mbps connection
+
+## Installation
+
+The newest binary releases can be found [here](). With the exception of the .exe and .dmg files, they are archives of the latest executable binaries for each release. Instructions are provided for macOS and Linux below:
+
+### macOS
+
+1. Download [coda.zip](https://s3-us-west-2.amazonaws.com/wallet.o1test.net/coda-daemon-macos.zip)
+2. Unzip anywhere, `cd` to navigate to the Coda directory
+3. Run `brew install miniupnpc` to install [MiniUPnP client](https://github.com/miniupnp/miniupnp)
+4. Set up port forwarding (see below)
+5. `./coda -help` to see if it works
+
+### Linux (Ubuntu / Debian)
+
+1. Download [coda.deb](https://s3-us-west-2.amazonaws.com/packages.o1test.net/pool/unstable/c/co/coda-testnet-postake-medium-curves_0.0.1-release-beta-0d13213e.deb)
+2. Run `apt-get install miniupnpc` to install [MiniUPnP client](https://github.com/miniupnp/miniupnp)
+3. Set up port forwarding (see below)
+4. Double click
```

Comment:
I would make the `.deb` installation step 2, after downloading.

Also, I would give the command-line installation command, since the port forwarding stuff is also via the command line. I think `dpkg -i the-deb-file` works across Debian/Ubuntu (and Mint, which I'm using).

Reply:
good call - I'll update this